apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainCheckerSamples.kt

```diff
@@ -27,4 +28,21 @@ class IterableLikeToContainCheckerSamples {
             }
         }
     }
+
+    @Test
+    fun atLeast() {
+        expect(listOf("A,B,C,A,B,B")).toContain.inAnyOrder.atLeast(3).entry{
+            toEqual("B")
+        }
+
+        expect(listOf(1,2,3,4,5,6,4)).toContain.inAnyOrder.atLeast(2).entry{
+            toEqual(4)
+        }
+
+        fails {
```

Comment:
Would be nice if you could explain why it fails
```suggestion
        fails { // because contains only one "A"
```